File: desafio-03/robsonalexandre/python/palindromos.py
#!/usr/bin/env python3
""" Desafio 03: Imprimir todos os números palindrômicos entre dois outros números """
import sys
def main():
    """ Inverte número e imprime se igual ao seu reverso. """
    for num in range(int(sys.argv[1]), int(sys.argv[2])):
        if int(str(num)[::-1]) == num:
            print(num)

if len(sys.argv) > 1:
    main()
else:
    print("É necessário informar 2 números:\n  palindromos.py NUM NUM")

File: desafio-03/PinheiroR90/javascript/palindromos.js

const ehPalindromo = (num) => {
    const str = num.toString();
    return str === str.split('').reverse().join('');
  }
  
  const encontrePalindromos = (start, end) => {
    const palindromes = [];
    for (let i = start; i <= end; i++) {
      if (ehPalindromo(i)) {
        palindromes.push(i);
      }encontrePalindromos
    }encontrePalindromos
    return palindromes;
  }
  
  const start = 1; 
  const end = 10000;
  const palindromos = encontrePalindromos(start, end);
  console.log(`Números palindrômicos entre ${start} e ${end}:`, palindromos);
  